api/graph/src/main/java/quarks/graph/Connector.java

```diff
@@ -115,5 +115,27 @@ Licensed to the Apache Software Foundation (ASF) under one
      * 
      * @return set of tag values.
      */
-    Set<String> getTags(); 
+    Set<String> getTags();
+    
+    /**
+     * Set the alias for the connector.
+     * <p>
+     * The alias must be unique within the topology.
+     * The alias may be used in various contexts:
+     * <ul>
+     * <li>Runtime control services for the connector are registered with this alias.</li>
```

Comment:
Sort of confusing since there are really 3 abstraction levels to the system :-)  Would changing "... the connector" => " the Connector" (talking about this class not connectors in the sense of mqtt, iot, ...) or "this Connector" help clarify?

[Maybe I'm just misreading your second sentence.  There's nothing stream/TStream specific about a control service nor ControlService. ]

At the API level users deal in terms of TStream.alias() - an alias for the stream and from their perspective control services are registered for the stream.

But at the graph level of the system, its a graph.Connector (the logical equivalent of an output port / stream) that has the alias and services are registered are for the Connector's alias.  It has no direct knowledge of TStream.

The oplet level, doesn't have any knowledge of TStream or Connector, it just deals in Consumer for an output port.  It needs to register the control service with the alias from the "output port" (aka stream / Connector / TStream).   I'm particularly interested in a sanity check on the addition of OutputContext to OpletContext.  It, or something else, is needed for the oplet to be able to get the alias to register mbean.